packages/headless/src/features/configuration/configuration-state.ts

```diff
@@ -99,7 +99,7 @@ export const getConfigurationInitialState: () => ConfigurationState = () => ({
   },
   analytics: {
     enabled: true,
-    apiBaseUrl: `${platformUrl()}${analyticsAPIEndpoint}`,
+    apiBaseUrl: `${analyticsUrl()}${analyticsAPIEndpoint}`,
```

Comment:
If someone configures a non-us-prod platform url, will the analytics url adjust so that it is in-sync?

Reply:
👀 

Good catch, I will need to do some modification in the configuration slice when someone manually configure those.